src/main/java/redis/clients/jedis/JedisClientConfig.java

```diff
@@ -80,4 +80,18 @@ default HostAndPortMapper getHostAndPortMapper() {
     return null;
   }
 
+  public interface ClientSetInfoConfig {
```

Comment:
Should we use a separate inner config for two fields? Or, do we add those two fields directly into existing interface?